pkg/osm/version/annotations.go

```diff
@@ -1,4 +1,4 @@
-package osm
+package version
```

Comment:
What's the reason behind this rename?
It seems that these are `osm` CRDs, no?

Reply:
@draychev - I had to move some stuff from `pkg/osm` to `pkg/osm/utils` and `pkg/osm/version` as that was the only way to resolve circular dependencies.

Reply:
@draychev - Let me know if these package renames are good to go or if there's another better way to resolve circular imports.